data/resources/ui/sidebar-row.ui

```diff
@@ -116,7 +117,6 @@
             <child>
               <object class="GtkLabel" id="unread_count_label">
                 <property name="valign">center</property>
-                <property name="ellipsize">end</property>
```

Comment:
Since we've found out that that's not the reason of the bug you can just leave it.